tests/intern.js

```diff
@@ -35,7 +36,7 @@ function (args, topic, firefoxProfile) {
     // A fully qualified URL to the Intern proxy
     proxyUrl: 'http://127.0.0.1:9090/',
 
-    asyncTimeout: 5000, // milliseconds
+    asyncTimeout: parseInt(asyncTimeout, 10), // milliseconds
```

Comment:
Maybe it's better to put the `parseInt` up on line 24 where asyncTimeout is set.